faf-commons-api/settings.gradle

```diff
@@ -1,2 +0,0 @@
-rootProject.name = 'faf-commons'
```

Comment:
Why remove that?

Reply:
Because it is a copy&paste error. This is a submodule and it is not faf-commons